### PR TITLE
Portfolio: mobile version PR

### PR DESCRIPTION
**Pull request by inakiesk**
This branch was dedicated to building the "works" section and the final "About" section, the changes are the following:
-Add html structure to works and about sections.
-Add styling to works and about sections, adjusting fonts and sizes.
-Add positioning to works and about sections.
-Add background to works section.
-Add listed icons in about section.
-Add buttons and buttons interactions to works and about section.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -37,6 +39,147 @@ <h2 class="connect">LET’S CONNECT</h2>
...
+      <ul class="program-list">
+        <li>html</li>
+        <li>css</li>
+        <li>javascript</li>
+      </ul>
+      <nav><input type="button" value="See Project" /></nav>
+    </div>
+  </article>
+
+  <article class="work2">
```
Comment (Kossi-stack) [inline index.html:64]: - [ ] Kindly fix the height of the card, the button see project is too close

Capturesee

File: index.html [hunk 2]
```diff
@@ -37,6 +39,147 @@ <h2 class="connect">LET’S CONNECT</h2>
...
+        <li>css</li>
+        <li>javascript</li>
+      </ul>
+      <nav><input type="button" value="See Project" /></nav>
+    </div>
+  </article>
+
+</section>
+
+<section class="about-section">
```
Comment (Kossi-stack) [inline index.html:129]: - [ ] Kindly style well work section and about section
Capturefix

**Review by Kossi-stack — CHANGES_REQUESTED** [review]
## Hi @,

**Good job so far! 👍🏻**
There are some issues that you still need to work on to go to the next project but you are almost there!💪

68747470733a2f2f6d656469612e67697068792e636f6d2f6d656469612f556461535559436c723932636a57706556782f67697068792e676966

## To Highlight

✅ Descriptive pull request message
✅ linter error checks validated
✅ Documented your work professionally
✅ README file is descriptive.
✅ Commit messages are meaningful.

## Required Changes ♻️

- [ ] Kindly edit the PR's title to something more descriptive. it can be "Portfolio: mobile version PR".
- [ ] Make sure to create a folder call images or assets then put all images and logo inside.

 _Check the comments under the review for more changes_.

## Optional suggestions

 - [x] Kindly add a descriptive pull request message, This helps other programmers who want to take a look at your code to understand what changes you made to a particular branch, you can find an example [here](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/documentation/articles/professional_repo_rules.md#5-basic-rules-to-follow-in-microverse-projects).

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

_**Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.**_

As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Review by vickymarz — APPROVED** [review]
Hi @inakiesk 

# 🟢  Status: APPROVED 🏆 🎉 🎊 

You have done a great job on this project, keep it up 👍 🥇. Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
congratulations!  🎉💯🌟 

## Top highlight 🎯

- The Pull Request is following the GitHub Flow ✅ 
- The PR title and description are professional and descriptive ✅ 
- The commit messages are meaningful ✅ /
- Linters are working properly ✅ 


## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**